bin/import.ml

```diff
@@ -161,9 +160,8 @@ let restore_cwd_and_execve (common : Common.t) prog argv env =
 
 (* Adapted from
    https://github.com/ocaml/opam/blob/fbbe93c3f67034da62d28c8666ec6b05e0a9b17c/src/client/opamArg.ml#L759 *)
-let command_alias cmd name =
-  let term, info = cmd in
-  let orig = Term.name info in
+let command_alias cmd term name =
```

Comment:
There's no public way to get `term` from `cmd`, so I passed it and exposed the term in `Build_cmd`.